docker-for-windows/index.md

```diff
@@ -307,75 +194,58 @@ access the mounted volumes.
 
 > Tips on shared drives, permissions, and volume mounts
 >
- * Shared drives are only required for mounting volumes in
- [Linux containers](#switch-between-windows-and-linux-containers), not for
- Windows containers. For Linux containers, you need to share the drive where the
- Dockerfile and volume are located. If you get errors such as `file not found`
- or `cannot start service` you may need to enable shared drives. See
- [Volume mounting requires shared drives for Linux containers](troubleshoot.md#volume-mounting-requires-shared-drives-for-linux-containers).)
+ * Shared drives are only required for mounting volumes in [Linux containers](#switch-between-windows-and-linux-containers),
+ not for Windows containers. For Linux containers, you need to share the drive
+ where the Dockerfile and volume are located. If you get errors such as `file not found`
+ or `cannot start service` you may need to enable shared drives. See [Volume mounting requires shared drives for Linux containers](troubleshoot.md#volume-mounting-requires-shared-drives-for-linux-containers).)
 >
-* If possible, avoid volume mounts from the Windows host, and instead mount on
-the MobyVM, or use a [data
-volume](/engine/tutorials/dockervolumes.md#data-volumes)
-(named volume) or [data
-container](/engine/tutorials/dockervolumes.md#creating-and-mounting-a-data-volume-container).
-There are a number of issues with using host-mounted volumes and network paths
-for database files. see the troubleshooting topic on [Volume mounts from
-host paths use a nobrl option to override database
-locking](troubleshoot.md#volume-mounts-from-host-paths-use-a-nobrl-option-to-override-database-locking).
+ * If possible, avoid volume mounts from the Windows host, and instead mount on
+ the MobyVM, or use a [data volume](/engine/tutorials/dockervolumes.md#data-volumes)
```

Comment:
I think we should avoid such a name (MobyVM), and keep a single way to denote it.  Below, there is "Linux VM", and I think that's better.  Or just "VM".

Reply:
wilco

Reply:
Just wondering ... @akimd, perhaps we should call it what it is ... `MobyLinuxVM`? The Internets seems to use that.

Reply:
Hi Gwen.  I've been discussing with the other guys from d4d, and the current status is:
- MobyVM is really a misnomer.  The name Moby was actually taken from it (which caused
  a lot of confusion) for some other project.  So we should definitely stay away from it.
- If we were to be precise, today it's LinuxKit VM.  But then again, in the future things
  might change, and we prefer to stick to the name of the feature/service (it's a Linux VM) rather
  than the name of the implementation.
- That being said, on Windows the name of the VM is exposed to the users when they
  deal with HyperV.  And to avoid making upgrades harder on the users, the original name
  was kept (MobyLinuxVM).  We might want to move away from this in the future.
  But only die hard users might see it.

So I suggest that we stick to Linux VM on both architectures.

Reply:
Thanks for the backstory, @akimd. Very helpful.  I ended up using Linux VM in the end.